editor/editor_node.cpp

```diff
@@ -510,6 +510,10 @@ void EditorNode::_update_from_settings() {
 	Viewport::MSAA msaa = Viewport::MSAA(int(GLOBAL_GET("rendering/anti_aliasing/quality/msaa_2d")));
 	scene_root->set_msaa_2d(msaa);
 
+	// 2D doesn't use a dedicated SubViewport like 3D does, so we apply it on the root viewport instead.
+	const bool use_debanding = GLOBAL_GET("rendering/anti_aliasing/quality/use_debanding");
+	scene_root->set_use_debanding(use_debanding);
```

Comment:
```suggestion
	scene_root->set_use_debanding(GLOBAL_GET("rendering/anti_aliasing/quality/use_debanding"));
```
It's not much longer this way.